elements/nuxeo-document-info-bar/nuxeo-document-info-bar.js

```diff
@@ -187,16 +178,7 @@ Polymer({
       <nuxeo-publication-info-bar document="[[document]]"></nuxeo-publication-info-bar>
     </template>
 
-    <!-- cold storage -->
-    <template is="dom-if" if="[[_contentStoredInColdStorage(document)]]">
-      <div id="coldStorageInfoBar" class="bar coldStorage">
-        <div class="layout horizontal center flex">
-          <iron-icon noink icon="nuxeo:coldstorage"></iron-icon>
-          <span class="storedInColdStorage">[[i18n('documentPage.content.storedInColdStorage')]]</span>
-          <span>[[i18n('documentPage.content.lowResolutionPreview')]]</span>
-        </div>
-      </div>
-    </template>
+    <nuxeo-slot name="DOCUMENT_INFOBAR" model="[[actionContext]]"> </nuxeo-slot>
```

Comment:
I think the following would be more consistent with the way we are currently naming other slots and elements:
```suggestion
    <nuxeo-slot name="DOCUMENT_INFO_BAR" model="[[actionContext]]"> </nuxeo-slot>
```
WDYT?